### PR TITLE
New package: palapeli-18.08.3

### PR DESCRIPTION
**Pull request by non-Jedi**
Adding KDE jigsaw puzzle game. Tested locally on x86-64 glibc with no visible issues.

### CODE REVIEW

File: srcpkgs/palapeli/template [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Template file for 'palapeli'
+pkgname=palapeli
+version=18.08.3
+revision=1
+build_style=cmake
+hostmakedepends=extra-cmake-modules
+makedepends="libkdegames-devel kdoctools"
+depends=qhull
```
Comment (maxice8) [inline srcpkgs/palapeli/template]: quote

Comment (non-Jedi) [reply srcpkgs/palapeli/template]: Done. Quoted hostmakedepends as well on assumption that that was the preferred style.

**Comment by Chocimier** [comment]
kdoctools qt5-qmake qt5-host-tools added to hostmakedepends should fix cross-compilation.

**Comment by maxice8** [comment]
On Sun, Dec 02, 2018 at 11:53:23PM -0800, Piotr wrote:
> kdoctools qt5-qmake qt5-host-tools added to hostmakedepends should fix cross-compilation.
> 
> -- 
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub:
> https://github.com/void-linux/void-packages/pull/5361#issuecomment-443619188

and kconfig

**Pull request closed without merging**